build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -446,11 +446,13 @@ The aim of this pull request is to sync ${source} and ${target} branch. This bra
 1. Acknowledge the pull request by adding a comment -- "Actively working on it".
 2. Merge ${mergeBranch} into the target branch, ${target}. **The direction of the merge matters!** You need to checkout ${target} and merge ${mergeBranch} into it, then fast-forward ${mergeBranch} to the merge commit. To do that use the following git commands:
   - \`git fetch --all\` -- this ensures your remote refs are updated
+  - \`git remote -v\` -- displays the list of remote repositories associated with your Git repository along with their corresponding URLs. You have to choose the remote repository associated to URL: **https://github.com/microsoft/FluidFramework.git**. **Please repalce your remote accordingly.**
+  Assuming ${remote} is the remote associated to **https://github.com/microsoft/FluidFramework.git**
```

Comment:
```suggestion
  - \`git remote -v\` -- displays the list of remote repositories associated with your Git repository along with their corresponding URLs. You have to choose the remote associated with the **microsoft/FluidFramework** repository. Change the remote name in these example commands if yours is not ${remote}.
```

Remotes can use SSH, so just refer to the repo name.